modules/appeals_api/lib/appeals_api/sidekiq_retry_notifier.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+
+module AppealsApi
+  class SidekiqRetryNotifier
+    WEBHOOK_URL = 'https://hooks.slack.com/services/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'
```

Comment:
This is a temporary slack webhook in a test Slack team I made. Still waiting for authorization on the Incoming Webhook for Lighthouse, and have to put this credential in the env.